### PR TITLE
Add a few finnish and french cities

### PR DESCRIPTION
**Pull request by endormi**
Fix Belarus city (Brest)

### CODE REVIEW

File: presets.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ var PRESETS = map[string]PresetLocations{
 	"singapore":          PresetLocations{"singapore"},
 	"latvia":             PresetLocations{"latvia", "latvija", "riga", "rīga", "kuldiga", "kuldīga", "ventspils", "liepaja", "liepāja", "daugavpils", "jelgava", "jurmala", "jūrmala"},
 	"romania":            PresetLocations{"romania", "bucharest", "cluj", "iasi", "timisoara", "craiova", "brasov", "sibiu", "constanta", "oradea", "galati", "ploesti", "pitesti", "arad", "bacau"},
-	"belarus":            PresetLocations{"belarus", "minsk", "brest,belarus", "grodno", "gomel", "vitebsk", "mogilev", "slutsk", "borisov", "pinsk", "baranovichi", "bobruisk", "soligorsk"},
+	"belarus":            PresetLocations{"belarus", "minsk", "brest", "grodno", "gomel", "vitebsk", "mogilev", "slutsk", "borisov", "pinsk", "baranovichi", "bobruisk", "soligorsk"},
```
Comment (lauripiispanen) [inline presets.go]:
```suggestion
	"belarus":            PresetLocations{"belarus", "minsk", "brest,belarus", "grodno", "gomel", "vitebsk", "mogilev", "slutsk", "borisov", "pinsk", "baranovichi", "bobruisk", "soligorsk"},
```

File: presets.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ var PRESETS = map[string]PresetLocations{
 	"estonia":            PresetLocations{"estonia", "eesti", "tallinn", "tartu", "narva", "pärnu"},
 	"denmark":            PresetLocations{"denmark", "danmark", "copenhagen", "aarhus", "odense", "aalborg"},
 	"portugal":           PresetLocations{"portugal", "lisbon", "lisboa", "braga", "porto", "aveiro", "coimbra", "funchal", "madeira"},
-	"france":             PresetLocations{"france", "paris", "marseille", "lyon", "toulouse", "nice", "nantes", "strasbourg", "montpellier", "bordeaux", "lille", "rennes", "reims"},
+	"france":             PresetLocations{"france", "paris", "marseille", "lyon", "toulouse", "nice", "nantes", "strasbourg", "montpellier", "bordeaux", "lille", "rennes", "reims", "rouen", "toulon", "le+havre", "grenoble", "dijon", "le+mans", "brest", "tours"},
```
Comment (lauripiispanen) [inline presets.go]:
```suggestion
	"france":             PresetLocations{"france", "paris", "marseille", "lyon", "toulouse", "nice", "nantes", "strasbourg", "montpellier", "bordeaux", "lille", "rennes", "reims", "rouen", "toulon", "le+havre", "grenoble", "dijon", "le+mans", "brest,france", "tours"},
```

File: presets.go (hunk 1 above)
Comment (endormi) [reply presets.go]: Ah, my bad!

**Comment by endormi** [comment]
It's resolved.

**Pull request merged**